third_party/libpg_query/grammar/statements/select.y

```diff
@@ -2446,11 +2446,21 @@ a_expr:		c_expr									{ $$ = $1; }
 					n->location = @2;
 					$$ = (PGNode *)n;
 				}
-			| a_expr LAMBDA_ARROW a_expr
+			| LAMBDA a_expr SINGLE_COLON a_expr
```

Comment:
Can we make this `LAMBDA name_list SINGLE_COLON a_expr` instead?

* There should not be a need to use an `a_expr` on the left-hand side of a lambda, it should be only names
* This would then allow us to match Python for multi-parameter lambdas (i.e. `lambda x,y: x + y` works without parentheses)